File: Programowanie Aplikacji Mobilnych/klasa 4/Java + Xml/2409/app/src/main/java/com/example/a24_09/MainActivity.java
package com.example.a24_09;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {
    Button nwdBtn, sumaBtn;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });

        nwdBtn = findViewById(R.id.nwdBtn);
        sumaBtn = findViewById(R.id.sumaBtn);

        nwdBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(getApplicationContext(), NwdActivity.class);
                startActivity(i);
            }
        });

        sumaBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(getApplicationContext(), sumaCyfrActivity.class);
                startActivity(i);
            }
        });
    }
}

File: Programowanie Aplikacji Mobilnych/klasa 4/Kotlin + Xml/13_01_1/app/src/main/java/com/example/a13_01/MainActivity.kt
package com.example.a13_01

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
        val et = findViewById<EditText>(R.id.editText)
        val btn = findViewById<Button>(R.id.button)

        btn.setOnClickListener {
            val uri: Uri = Uri.parse(String.format("geo:0,0?q=%s",et.getText()))
            val i = Intent(Intent.ACTION_VIEW, uri)
            i.setPackage("com.google.android.apps.maps")
            startActivity(i)
        }
    }
}